snowplow/iglu-client-embedded/schemas/com.metabase/simple_event/jsonschema/1-0-0

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Generic event to track interactions and actions that happen within Metabase. This should be used as a default schema for tracking events. If more specific context is needed, we can use custom context entities (https://docs.snowplow.io/docs/understanding-your-pipeline/entities/).",
+  "self": {
+    "vendor": "com.metabase",
+    "name": "simple_event",
```

Comment:
this line is the only thing I changed, weird git didn't understood i was a rename

Reply:
Might be because you changed it in one go and not with 2 commits or something.